### PR TITLE
[PYTHON-311] Test for inserting collections of UDTs

### PR DESCRIPTION
**Pull request by GregBestland**

### CODE REVIEW

File: tests/integration/cqlengine/model/test_udts.py [hunk 1]
```diff
@@ -286,3 +286,43 @@ class AllDatatypesModel(Model):
 
         for i in range(ord('a'), ord('a') + 15):
             self.assertEqual(input[chr(i)], output[chr(i)])
+
+    def test_nested_udts_inserts(self):
+        """
+        Test for inserting collections of user types using cql engine.
+        test_nested_udts_inserts Constructs a model that contains a list of usertypes. It will then attempt to insert
```
Comment (kishkaru) [inline tests/integration/cqlengine/model/test_udts.py]: There should be a newline between the brief description and the full description.

File: tests/integration/cqlengine/model/test_udts.py [hunk 2]
```diff
@@ -286,3 +286,43 @@ class AllDatatypesModel(Model):
...
+
+        @since 2.6.0
+        @jira_ticket PYTHON-311
+        @expected_result. No syntax exception thrown
+
+        @test_category data_types:udt
+        """
+
+        # This combination of model, and UT produces a syntax error in 2.5.1 due to improper quoting around
+        # the names collection
```
Comment (kishkaru) [inline tests/integration/cqlengine/model/test_udts.py]: Not sure why this inline comment is here. If this info is needed, it should be in the comment-block description.

**Comment by GregBestland** [comment]
Comments should be addressed, with latest push. Commits squashed.

File: tests/integration/cqlengine/model/test_udts.py [hunk 3]
```diff
@@ -286,3 +286,43 @@ class AllDatatypesModel(Model):
...
+        Test for inserting collections of user types using cql engine.
+
+        test_nested_udts_inserts Constructs a model that contains a list of usertypes. It will then attempt to insert
+        them. The expectation is that no exception is thrown during insert. For sanity sake we also validate that our
+        input and output values match. This combination of model, and UT produces a syntax error in 2.5.1 due to
+        improper quoting around the names collection.
+
+        @since 2.6.0
+        @jira_ticket PYTHON-311
+        @expected_result. No syntax exception thrown
```
Comment (kishkaru) [inline tests/integration/cqlengine/model/test_udts.py]: Extra . here.

**Comment by kishkaru** [comment]
LGTM.

**Pull request merged**